scripts/gbmake

```diff
@@ -296,11 +322,11 @@ touch "$GNUBEE_KERNEL_OBJECTS/initramfs-files.txt"
 case $1 in
   defconfig* | *defconfig ) gnubee_defconfig $1;;
   chroot ) gnubee_chroot ;;
-  initramfs ) gnubee_initramfs -f;;
-  modules ) gnubee_modules ;;
+  initramfs ) gnubee_initramfs $2 -f;;
```

Comment:
The removal of the space char before the double semicolon is inconsistent with the untouched lines, don't do it, or do it to all the lines.